### PR TITLE
fixed results count calculated without search settings

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

File: Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs [hunk 1]
```diff
@@ -121,6 +125,7 @@ async Task<List<AccommodationAvailabilityResult>> Convert(EdoContracts.Accommoda
                     .ToList();
 
                 var duplicates = await _duplicatesService.GetDuplicateReports(supplierAccommodationIds);
+                var searchSettings = await _accommodationBookingSettingsService.Get(agent);
```
Comment (mingazhev) [inline Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs:128]: Why not getting this once and passing settings as method param?

Comment (DFSko) [reply Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs]: Fixed

File: Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs [hunk 2]
```diff
@@ -28,14 +28,16 @@ public class WideAvailabilitySearchTask
...
+            ILogger<WideAvailabilitySearchTask> logger,
+            IAccommodationBookingSettingsService accommodationBookingSettingsService)
         {
             _storage = storage;
             _priceProcessor = priceProcessor;
             _duplicatesService = duplicatesService;
             _supplierConnectorManager = supplierConnectorManager;
             _dateTimeProvider = dateTimeProvider;
             _logger = logger;
+            _accommodationBookingSettingsService = accommodationBookingSettingsService;
```
Comment (mingazhev) [inline Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs:40]: Is it still used?

**Review by mingazhev — CHANGES_REQUESTED** [review]

**Review by mingazhev — COMMENTED** [review]
Looks like unused field left

File: Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs (hunk 2 above)
Comment (DFSko) [reply Api/Services/Accommodations/Availability/Steps/WideAvailabilitySearch/WideAvailabilitySearchTask.cs]: Removed

**Review by mingazhev — APPROVED** [review]

**Pull request merged**